pkg/translate/translate.go

```diff
@@ -0,0 +1,57 @@
+package translate
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/convert"
+	"github.com/expect-digital/translate/pkg/model"
+	pb "github.com/expect-digital/translate/pkg/server/translate/v1"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type TranslateServiceServer struct {
+	pb.UnimplementedTranslateServiceServer
+}
+
+func New() *TranslateServiceServer {
+	return new(TranslateServiceServer)
+}
+
+func (t *TranslateServiceServer) DownloadTranslationFile(
+	ctx context.Context,
+	req *pb.DownloadTranslationFileRequest,
+) (*pb.DownloadTranslationFileResponse, error) {
+	if len(req.GetTranslationId()) == 0 {
+		return nil, status.Errorf(codes.InvalidArgument, "Missing param translation_id")
+	}
+
+	var (
```

Comment:
The same way I advised [here](https://github.com/expect-digital/translate/pull/24/files#r1061771670)